### PR TITLE
SPACON-430 SPACON-431: Adding the feature and opertion protobufs.

### PR DESCRIPTION
**Pull request by boundlessgeo-yancy**

### CODE REVIEW

File: Feature.proto [hunk 1]
```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+option java_package = "com.boundlessgeo.spatialconnect.schema";
+option java_outer_classname = "SpatialConnect";
+
+message Feature {
+  string fid = 1;
+  map<string, string> properties = 2;
+  string geometryType = 3;
```
Comment (boundlessgeo-yancy) [inline Feature.proto]: We can remove this since the type is included in the geometry field.

File: Feature.proto [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  bytes geometry = 4;
+};
+
+message Operation {
+  OperationType operation = 1;
+  string layer = 2;
```
Comment (boundlessgeo-yancy) [inline Feature.proto]: Rename to source.

Comment (boundlessgeo-yancy) [inline Feature.proto]: Source could be workspace.layer, device.workspace.layer, etc.

**Comment by mrcnc** [comment]
👍

**Pull request merged**